index.html

```diff
@@ -11,34 +11,217 @@
+                            </ul>  
+                        </div>
+                        <div class="all-cards">            
+                            <div class="no-btn">
+                                <h1>Profesional Art Printing Data</h1>
+                                <p>
+                                   A daily selection of privately personalized reads; no accounts or sign-ups required. 
+                                   has been the industry's standard
+                                </p>
+                                <ul class="cards-lists">
+                                  <li><a href="#" class="html-btn">html</a></li>
+                                  <li><a href="#" class="boot-btn">bootstrap</a></li>
+                                  <li><a href="#" class="ruby-btn">Ruby</a></li>
+                                </ul>
+                            </div>
+                            <ul>
+                                <a href="#"><button class="some-pro-text hover-effect">See Project</button></a>
+                            </ul>  
+                        </div>
+                        <div class="all-cards">            
+                            <div class="no-btn">
+                                <h1>Profesional Art Printing Data</h1>
+                                <p>
+                                   A daily selection of privately personalized reads; no accounts or sign-ups required. 
+                                   has been the industry's standard
+                                </p>
+                                <ul class="cards-lists">
+                                  <li><a href="#"class="html-btn">html</a></li>
+                                  <li><a href="#" class="boot-btn">bootstrap</a></li>
+                                  <li><a href="#" class="ruby-btn">Ruby</a></li>
+                                </ul>
+                            </div>
+                            <ul>
+                                <a href="#"><button class="some-pro-text hover-effect">See Project</button></a>
+                            </ul>  
+                        </div>
+                    </div>  
+                </div>
+            </div>
+        </div>
+        <section class="about-container">
+            <div class="first-about-container">
+                <h1 class="about-header">About me</h1>
+                <p class="about-para">Hello I’m a software developer! I can help you build a product , 
+                    feature or website Look through some of my work and experience! 
+                    If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+                <a href="#"><button class="about-btn-text hover-effect">Get My Resume</button></a>
+                <img class="about-illu" src="./img/about/illustration-about-me-mobile.svg" alt="illustration">
```

Comment:
Please consider adding a back-ground image to the section rather than adding it as plain `img tag`.


## Reason:--

Decorations are always backgrounded images, not img tags


## Solution:--

You can just use the CSS background-image property to add it to the section.